GPT/gpt.talon

```diff
@@ -5,7 +5,7 @@
 #   Example: `model fix grammar below` -> Fixes the grammar of the selected text and pastes below
 #   Example: `model explain this` -> Explains the selected text and pastes in place
 #   Example: `model fix grammar clip to browser` -> Fixes the grammar of the text on the clipboard and opens in browser`
-{user.model} <user.modelPrompt> [{user.modelSource}] [{user.modelDestination}]:
+{user.model} <user.modelPrompt> [{user.modelSource}] [{user.modelDestination}]$:
```

Comment:
Added this. Let me know if you disagree but I think there are few situations in which a user will want to chain the model command since they haven't seen the output until it is returned. I think $ prevents a lot of the weird situations where talon thinks there is a second command inside of this.

This still allows for prefix chaining before the command

Reply:
I was thinking about going further and including the ^ before the start of the command. I didn't understand what prefix chaining is-can you give me an example?

Reply:
When I say prefix chaining I just mean chaining another talon command before. So for instance you could use a cursory list command before and then chain with the model as long as it has at the end of the chain

ie 'take block this model fix grammar'

All as one sentence. That is something I sometimes use.

Reply:
Ah I see. In that particular case I suppose you could use the cursorless beta command too (I.e. model fix grammar block) but yeah I can see what you mean.

Reply:
Yup, although for arbitrary commands that don't fit in the grammar then it becomes a little slower to use. I don't think anything is disrupted by not having the anchor at the start so I think we can just have the anchor at the end to prevent chaining after. The main issue is having the ability to chain after user.text. (Sometimes Talon does not know if the user is trying to dictate raw text or break out of that with a separate command)

The $ end anchor removes that so it is probably sufficient just like this